Package.swift

```diff
@@ -153,7 +155,7 @@ let package = Package(
         .lambdaTarget(
             name: "AutoPings",
             additionalDependencies: [
-                .product(name: "SotoS3", package: "soto"),
+                .product(name: "SotoS3", package: "soto")
```

Comment:
Why? Trailing commas in array and dictionary literals are considered good form.

Reply:
Not sure how but swift-format somehow ran on a bunch of files and automatically formatted them for me

Reply:
For the most part I actually find the changes to be a marked improvement, but in this particular case I'd say a rule needs adjusting

Reply:
Wait now that i'm looking at it from PC...
This is actually completely expected formatting behavior. If there is only 1 element, it excludes the trailing comma.
Otherwise all elements including the last element will have a trailing comma in a collection.

Reply:
Still bad form IMO

Reply:
I can't see a settings for disabling that, but I don't mind it anyway.